src/Robo/Plugin/Commands/CICommands.php

```diff
@@ -142,7 +142,7 @@ public function jobFixCodingStandards(): Result
      */
     protected function getCustomCodePaths(): array
     {
-        if (!$customCodePaths = Robo::config()->get('custom_code_paths')) {
+        if (!is_array($customCodePaths = Robo::config()->get('custom_code_paths'))) {
```

Comment:
Ok. All my comments assume control of the called function, so they can be ignored. It's too bad the returns are vague.

Reply:
> It's too bad the returns are vague.

With my testing I think the returns are actually _not vague_:

> The whole purpose of this method is to validate that we get an array back but if I just called Robo::config()->get('custom_code_paths')) [here](https://github.com/ChromaticHQ/usher/blob/phpstan-strict/src/Robo/Plugin/Commands/CICommands.php#L53) we would still get an exception for passing a non-array to implode() (I checked).

I am wondering if we can remove this defensiveness because these calls will already return exceptions if the value is not present. 🤔

Reply:
After further testing, I think the error messages we are able to provide with these extra checks are more descriptive and helpful since we know more specifically what we are expecting so I would advocate for leaving them.